<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>history</class>
 <widget class="QMainWindow" name="history">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>search by nickname</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>search by nickname keyword</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>search by content keyword</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>search by date</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>search by date period</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_search">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>search</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_endDate">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_to">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Devanagari</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>~</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>421</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
